src/lib/pages/evm-tx-details/components/evm-event-box/index.tsx

```diff
@@ -0,0 +1,160 @@
+import { Flex, Stack, Text } from "@chakra-ui/react";
+import { Fragment, useState } from "react";
+import { ExplorerLink } from "lib/components/ExplorerLink";
+import { CustomIcon } from "lib/components/icon";
+import { LabelText } from "lib/components/LabelText";
+import { Tooltip } from "lib/components/Tooltip";
+import { TypeSwitch } from "lib/components/TypeSwitch";
+import type { TxReceiptJsonRpcLog } from "lib/services/types";
+import type { EvmVerifyInfo, Nullable } from "lib/types";
+import { evmParseLog } from "lib/utils";
+import { EvmEventBoxData } from "./evm-event-box-data";
+import { EvmEventBoxTopics } from "./evm-event-box-topics";
+import { EvmEventBoxTabs } from "../../types";
+
+interface EvmEventBoxProps {
+  log: TxReceiptJsonRpcLog;
+  evmVerifyInfo: Nullable<EvmVerifyInfo>;
+}
+
+export const EvmEventBox = ({ log, evmVerifyInfo }: EvmEventBoxProps) => {
+  const [currentTab, setCurrentTab] = useState(EvmEventBoxTabs.Hex);
+  const parsedLog = evmParseLog(evmVerifyInfo?.abi ?? [], log);
```

Comment:
Should we allow the parsedLog variable to be either LogDescription or undefined?